<launch>
    <arg name="robot_name1" default="robot1"/>
    <arg name="robot_name2" default="robot2"/>
    <node pkg="robot_action_mng" type="robot1_action_mng" name="$(arg robot_name1)_action_mng" output="screen">
        <param name="move_base_node1" value="/$(arg robot_name1)/move_base"/>
        <param name="map_frame1" value="$(arg robot_name1)/map"/>
    </node>
    <node pkg="robot_action_mng" type="robot2_action_mng" name="$(arg robot_name2)_action_mng" output="screen">
        <param name="move_base_node2" value="/$(arg robot_name2)/move_base"/>
        <param name="map_frame2" value="$(arg robot_name2)/map"/>
    </node>
</launch>